Dockerfile

```diff
@@ -7,7 +7,7 @@ COPY pyproject.toml LICENSE README.md ./
 
 # Install dependencies
 RUN apt-get update && \
-  apt-get install -y procps && \
+  apt-get install -y git procps && \
```

Comment:
Why do we need git in the docker container? Should we add a new development layer?

Reply:
As long as we are not running pre-commit, we do not need `git`, no. I'll remove it.